### PR TITLE
{vis}[foss/2020b] Nipype v1.6.0 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)
Depends on ~#12890~ for NiBabel.

### CODE REVIEW

**Comment by verdurin** [comment]
@akesandgren could you look at the unit test error please.

File: easybuild/easyconfigs/n/Nipype/Nipype-1.6.0-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+dependencies = [
+    ('Python', '3.8.6'),
+    ('lxml', '4.6.2'),
+    ('networkx', '2.5'),
+    ('NiBabel', '3.2.1'),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/n/Nipype/Nipype-1.6.0-foss-2020b.eb:21]: To fix `AssertionError: 'source_urls' should not be defined in exts_default_options when using the default value in Nipype-1.6.0-foss-2020b.eb`:
```suggestion
```

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
zen2-rockylinux-88 - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/SebastianAchilles/c1df8e27bea343d6bdd713259e346ce8 for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3147.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/09030b7ef30dbfc38463dd51aced567a for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**